Source/DafnyCore/Prelude/Sequences.dfy

```diff
@@ -0,0 +1,796 @@
+module {:extract} Sequences {
+  export
+    provides Lists, Boxes
+    provides Seq, Length, AboutLength
+    provides Empty, LengthEmpty0, LengthEmpty1
+    provides Index, Append, IndexAppend
+    provides Build, BuildInv0, BuildInv1, BuildInjective, LengthBuild
+    provides Update, IndexUpdate, LengthUpdate
+    provides Contains, SeqContainsItsElements, EmptyContainsNothing, AppendContains, BuildContains
+    provides Equal, AboutEqual, Extensionality
+    provides SameUntil, AboutSameUntil
+    provides Take, LengthTake, IndexTake
+    provides Drop, LengthDrop, IndexDrop0, IndexDrop1
+    provides TakeContains, DropContains, AppendTakeDrop
+    provides DropNothing, TakeNothing, DropDrop
+    provides TakeUpdate0, TakeUpdate1, DropUpdate0, DropUpdate1, DropBuild
+
+  import opened Lists
+  import opened Boxes
+
+  // boogie: type Seq;
+  type {:extract_name "Seq"} Seq = List<Box>
```

Comment:
```suggestion
  type {:boogie_name "Seq"} Seq = List<Box>
```

Reply:
And so on, elsewhere.

Reply:
I changed `:extract_name` to `:extract_boogie_name`, and changed (the unused) `:extract` to `:extract_boogie`. I also modified the source to define these as constants in `BoogieExtractor.cs`, so they can easily be renamed in the future.